packages/react-native-session-replay/ios/Tests/RCTTextViewRecorderTests.swift

```diff
@@ -29,8 +29,7 @@ internal class RCTTextViewRecorderTests: XCTestCase {
         layerCornerRadius: CGFloat(1.0),
         alpha: CGFloat(1.0),
         isHidden: false,
-        intrinsicContentSize: CGSize(width: 100.0, height: 100.0),
-        overrides: .init(UIView())
```

Comment:
Init was removed on [2.22.1](https://github.com/DataDog/dd-sdk-ios/releases/tag/2.22.1), [here](https://github.com/DataDog/dd-sdk-ios/pull/2182/files#diff-a089b41a313632711a8fcec77119158ef0594187b4a469fbc1a38a025e0968c9L33-L38), so it's not a valid override anymore.